javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var names = [];
+  for (var x = 0; x < students.length; x++) {
+    for (var keys in students[x]) {
+      names.push(students[x][keys]);
+    }
```

Comment:
You only need one property, the `name` property. This would save every property's value. You don't need a loop for this portion, you just need to retrieve the value of the `name` property.